Management/Clients/AirQualityDataClient.cs

```diff
@@ -164,27 +166,27 @@ public AirQualityDataClient(IConfiguration configuration)
                 return (city, 100);
             }
 
-            if (aqius >= 51 && aqius <= 100)
+            if (aqius >= 50 && aqius <= 100)
```

Comment:
why you are changing the edge case here? 

So now 100, 150, 200 and 300 will be a overlap case, which is what I tried to avoid

Reply:
Restored.